server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/api.ts

```diff
@@ -48,6 +48,8 @@ export function create(
 		throttleIdPrefix: (req) => getParam(req.params, "tenantId"),
 		throttleIdSuffix: Constants.alfredRestThrottleIdSuffix,
 	};
+    const tenantThrottler = tenantThrottlers.get(Constants.generalRestCallThrottleIdPrefix);
```

Comment:
let's call this generalTenantThrottler to avoid confusion

Reply:
Done. :)